File: SkyPayment.Shared/Menu/MenuResponseModel.cs
using System.Collections.Generic;


namespace SkyPayment.Shared
{
    public class MenuResponseModel
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public ICollection<MenuItemResponse> Items { get; set; }
        public string ManagementUserId { get; set; }
        public IEnumerable<string> RestaurantId { get; set; }

    }


}

File: SkyPayment.UI/skypayment-management-ui/src/api-models/menu/MenuResponseModel.ts
export interface MenuResponseModel {
    id: string;
    name: string;
    items: MenuItemResponse[];
    managementUserId: string;
    restaurantId: string[];
}
export interface MenuItemResponse {
    id: string;
    name: string;
    price: number;
    ingredients: string;
    productContent: ProductContent;
    imageUrl: string;
    isActive: boolean;
}

export enum ProductContent {
    IsVegan = 1,
    IsDiabetic = 2,
    IsGlutenFree = 4,
    IsVegetarian = 8
}
